cmd/crossplane/core/core.go

```diff
@@ -196,6 +200,11 @@ func (c *startCommand) Run(s *runtime.Scheme, log logging.Logger) error { //noli
 		return errors.Wrap(err, "cannot create manager")
 	}
 
+	eb.StartLogging(func(format string, args ...interface{}) {
+		log.Info(fmt.Sprintf(format, args...))
```

Comment:
Info or debug? 🤔

Reply:
I want to see when we are hotlooping. That we only get from `Info`.

Reply:
Or another argument: considering logs lower-level than events, if we see too many event logs, we very clearly have too many kube events too 😄

Reply:
I am also concerned that it could be too verbose for info. This is also consistent with current manual logging behavior.

Reply:
why too verbose?

> considering logs are lower-level than events

If this is too verbose for the logs, we should not create that many events in the first place. Events are end-user-facing.

Reply:
Please leave these as debug in this PR. We're due for a discussion on our approach to logging in this project, but I don't want to conflate that with this otherwise small and valuable change.